.github/workflows/CI.yaml

```diff
@@ -44,22 +44,21 @@ jobs:
 
     # More info on options: https://github.com/conda-incubator/setup-miniconda
     # An environment for the minimum versions 
-    - uses: conda-incubator/setup-miniconda@v2
+    - name: setup micromamba
+      uses: mamba-org/setup-micromamba@v1
       with:
-        python-version: ${{ matrix.python-version }}
         environment-file: devtools/conda-envs/test_${{ matrix.env_file }}.yaml
-        channels: conda-forge, defaults
-        activate-environment: test
-        auto-update-conda: false
-        auto-activate-base: false
-        show-channel-urls: true
-    
+        environment-name: test
+        create-args: >-
+          python=${{ matrix.python-version }}
+
     - name: Install package
       # conda setup requires this special shell
       shell: bash -l {0}
       run: |
         python -m pip install . --no-deps
-        conda list
+        pip list
+        micromamba list
```

Comment:
```suggestion
```
Is this necessary? As far as I can tell, `micromamba list` is run for the `setup micromamba` step already.

Reply:
I'm keeping the spirit of what the original CI file did (there was a conda list call here).

Generally having a step post install that tells you everything in the environment is a good idea though - it tells you if anything has changed through pip override

Reply:
I see, good point!